pytripgui/canvas_vc/plotter/managers/plotting_manager.py

```diff
@@ -0,0 +1,115 @@
+from pytripgui.canvas_vc.plotter.bars import BarProjection
+from pytripgui.canvas_vc.plotter.coordinate_info import CoordinateInfo
+from pytripgui.canvas_vc.plotter.images import CtxImage, DoseImage, LetImage
+from pytripgui.canvas_vc.plotter.managers import PlacementManager
+
+
+class PlottingManager:

```

Comment:
Add a comment describing responsibility of this class

Reply:
Done